spaghetti-kotlin-support/src/main/groovy/com/prezi/spaghetti/kotlin/KotlinEnumGeneratorVisitor.groovy

```diff
@@ -36,7 +36,7 @@ ${values.join("\n")}
 		}
 
 		fun valueOf(name:String):${enumName} {
-			var result:MyEnum
+			var result:${enumName}
```

Comment:
Yes. Somebody already commented on this. I had to rebase afterwards, just to lose those comments. I WANT this to sink into oblivion, as an untagged commit, okay? So: no, we are not going to merge this pull, ever.